docs/modules/ROOT/pages/reference/service-requirements.adoc

```diff
@@ -0,0 +1,85 @@
+= Service Requirements and Iterations
+
+Implementing a service is iterative work.
+The idea is to release early and release often, until the desired functionality, stability and maturity is reached.
+
+This page gives an idea on the iterative steps.
+Except the initial version and the final version, all steps in-between can be in any order, as it fits best.
+
+Important is that every step already has the desired quality.
```

Comment:
I'm wondering:

What about external services where we may not be able to cover all the steps? Also for converged services on the cluster. What if implementing maturity "Marbas" would be so much effort for a specific service that it may not be worth it to implement? Should we allow some leeway for each service in respect to the requirements?

Reply:
I've added a few sentences to incorporate this concern. What do you think about it now?

Reply:
I would call it `desired quality` but maybe focus on the fact, that a step doesn't imply the implementation of another step. Maybe something like this:

```suggestion
Important is that every step is independent of each other. A service that implements `Abigor` doesn't necessarily already implement `Melchom `.
```